server/src/main/java/com/epam/aidial/core/server/security/ApiKeyStore.java

```diff
@@ -114,7 +118,7 @@ public Future<Boolean> invalidatePerRequestApiKey(ApiKeyData apiKeyData) {
         String apiKey = apiKeyData.getPerRequestKey();
         if (apiKey != null) {
             ResourceDescriptor resource = toResource(apiKey);
-            return vertx.executeBlocking(() -> resourceService.deleteResource(resource, EtagHeader.ANY), false);
+            return vertx.executeBlocking(() -> resourceOperationService.deleteResource(resource, EtagHeader.ANY), false);
```

Comment:
Please do not mess it up. Let's only delete prompts/conversations/files/applications with it. Other things are localized and should be left untouched.

Reply:
fixed